deployment/provision_vagrant_vm.sh

```diff
@@ -36,9 +36,17 @@ service redis-server restart
 # install apache
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
-# With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+OWNER=$(stat -c %u "$MOUNTPOINT/evap")
+
+# if, for any reason, vagrant failed to mount evap under the correct uid, use bindfs to link it to /opt/evap using the wanted user id
+if [ $OWNER != 1042 ]; then
+  apt-get -q install -y bindfs && \
+  mkdir -p "$REPO_FOLDER"
+  bindfs --map=$OWNER/1042:@$OWNER/@1042 "$MOUNTPOINT" "$REPO_FOLDER"
+  echo "sudo bindfs --map=$OWNER/1042:@$OWNER/@1042 '$MOUNTPOINT' '$REPO_FOLDER'" >> /home/$USER/.bashrc
+else
+  ln -s "$MOUNTPOINT" "$REPO_FOLDER"
```

Comment:
If we move this here, we should also remove the copy down in line 60

(and I'd also prefer to move the comment up to this part)

Reply:
like this?

Reply:
Yes :+1: